pkgs/by-name/de/dependency-track/package.nix

```
@@ -0,0 +1,110 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  jre_headless,
+  cyclonedx-cli,
+  makeWrapper,
+  maven,
+  nix-update-script,
+  nixosTests,
+}:
+let
+  version = "4.11.5";
+in
+
+maven.buildMavenPackage rec {
+  inherit version;
+  pname = "dependency-track";
+
+  src = fetchFromGitHub {
+    owner = "DependencyTrack";
+    repo = "dependency-track";
+    rev = version;
+    hash = "sha256-iVJidl1NbgTj5+DiU8MKJy0lMo608cfA2oMcrypLadI=";
+  };
+
+  patches = [
+    ./remove-frontend-download.patch
+    ./add-junixsocket.patch
+  ];
```

Prefix the patches with numbers, so that they are better viewed in the file manager and GitHub:

```suggestion
  patches = [
    ./0000-remove-frontend-download.patch
    ./0001-add-junixsocket.patch
  ];
```